handlers/github_api_handler.js

```diff
@@ -0,0 +1,141 @@
+const _ = require('lodash')
+const githubAPI = {
+  /**
+   * Removes duplicate items (repositories) from an array by creating a set from the array
+   * and then converting it back into an array. (Since sets can't contain duplicates)
+   * @param {Array} repoSet - A set of objects containing repository information
+   * @returns {Array} A unique set of objects containing repository information
+   */
+  removeDuplicates: function (repoSet) {
+    return _.uniqWith(repoSet, _.isEqual)
+  },
+  /**
+   * Gets a repository on github by its id and checks if the owner
+   * of the repo is the current user
+   * @param {Object} github - Allows for API calls to be made to the github api server
+   * @param {Object} repo_information - The returned object containing information on repositories contributed to by the user
```

Comment:
Naming conventions not being respected. `repo_information` is not correct. I am going to stop reviewing this PR until you can verify your linting is working properly.